lib/krane/cluster_resource_discovery.rb

```diff
@@ -66,7 +73,7 @@ def api_paths
 
     def fetch_api_path(path)
       @api_path_cache[path] ||= begin
-        raw_json, err, st = kubectl.run("get", "--raw", path, attempts: 2, use_namespace: false)
+        raw_json, err, st = kubectl.run("get", "--raw", cluster_url, attempts: 2, use_namespace: false)
```

Comment:
```suggestion
        raw_json, err, st = kubectl.run("get", "--raw", "#{cluster_url}/#{path}, attempts: 2, use_namespace: false)
```

Reply:
right, seems more consistent. ;) As i said, definitely not a ruby-pro.